proc/threads.c

```diff
@@ -841,6 +876,115 @@ int proc_threadCreate(process_t *process, void (*start)(void *), unsigned int *i
 }
 
 
+static unsigned int _proc_lockGetPriority(lock_t *lock)
+{
+	unsigned int priority = sizeof(threads_common.ready) / sizeof(threads_common.ready[0]) - 1;
```

Comment:
I agree with you that it is compile-time computation but for me it would be more readable to use one define instead of writing each time - `sizeof(threads_common.ready) / sizeof(threads_common.ready[0])` what is prone to mistakes.